src/main/java/org/fundacionjala/coding/abel/StopGninnipsMySdrow.java

```diff
@@ -0,0 +1,29 @@
+package org.fundacionjala.coding.abel;
+
+/**
+ *
+ */
+public class StopGninnipsMySdrow {
+    /**
+     *
+     */
+    private static final int MIN = 5;
+
+    /**
+     *
+     */
+
+
+    /**
+     * @param sentence string of input
+     * @return value return
+     */
+    public String spinWords(final String sentence) {
+        String[] string = sentence.split(" ");
+        String a = "";
+        for (int i = 0; i < string.length; i++) {
+            string[i] = string[i].length() >= MIN ? new StringBuilder(string[i]).reverse().toString() : string[i];
+        }
+        return a.join(" ", string);
```

Comment:
Don't use an unnecessary string variable to join an array.
You can use the String class
```java
return String.join(" ", string);
```